packages/core/graph/src/Graph.js

```diff
@@ -254,14 +254,7 @@ export default class Graph<TNode, TEdgeType: number = 1> {
     if (this.rootNodeId == null) {
       // If the graph does not have a root, and there are inbound edges,
       // this node should not be considered orphaned.
-      // return false;
-      for (let [, inboundNodeIds] of this.inboundEdges.getEdgesByType(nodeId)) {
-        if (inboundNodeIds.size > 0) {
-          return false;
-        }
-      }
-
-      return true;
+      return false;
```

Comment:
Needed this since "IdealGraph" may contain multiple islands which represent bundle groups. They may only contain 1 bundle, but this is not an orphan